File: 408A-line-to-cashier.py
n = int(input())
ppl = list(map(int, input().strip().split()))[:n]
time = []
for i in range(0, n):
    temp = 15 * ppl[i]
    product = list(map(int, input().strip().split()))[:ppl[i]]
    for j in range(0, ppl[i]):
        temp += product[j]*5
    time.append(temp)
time.sort()
print(time[0])

File: 408A-line-to-cashier.cpp
#include<iostream>
#include<algorithm>
using namespace std;
int main()
{
    int n, i, j, *ppl, *time, p, temp;
    cin >> n;
    ppl = (int *) calloc(n, sizeof(int));
    time = (int *) calloc(n, sizeof(int));
    for(i=0; i<n; i++)
    {
        cin >> ppl[i];
    }
    for(i=0; i<n; i++) 
    {
        temp = 15 * ppl[i];
        for(j=0; j<ppl[i]; j++) 
        {
            cin >> p;
            temp += (p*5);
        }
        time[i] = temp;
    }
    sort(time, time+n);
    cout << time[0] << endl;
    return 0;
}